OracleHTTPServer/README.md

```diff
@@ -95,6 +60,31 @@ _This volume will be created in "/var/lib/docker" directory or the location wher
 
 ######Later you can login to running container and configure Weblogic Server proxy plugin file and run *restartOHS.sh* script.
 
+### Stopping and re-starting OHS instance
+To stop the OHS instance, execute the following steps:
+1. Exec into the container and execute /bin/bash (Assuming the name of container is 'ohs')
```

Comment:
You should never require a user to `exec` into a running container. If you want to stop the OHS instance, you should write handlers for `docker stop` instead.